Fix internal spec links (#85)
awslabs/aws-encryption-sdk-specification: data-format/message-body.md

## Overview

The message body is a component of the [message](message.md).    
The message body contains the encrypted data, called the [encrypted content](#encrypted-content).    
The purpose of the message body is to define the structure containing the [encrypted content](#encrypted-content).

The structure of the body depends on the content type:

> 

Reviewer: Juuust out of range of the comment space again, but the links for framed and non-framed data are swapped.

The length of the encrypted content.    
The length MUST NOT be greater than `2^36 - 32`, or 64 gibibytes (64 GiB), 
due to restrictions imposed by the [implemented algorithms](../framework/algorithm-suites.md).

#### Encrypted Content

> The encrypted data as returned by the [encryption algorithm](../client-apis/encrypt.md).

Reviewer: ```suggestion
The encrypted data as returned by the [encryption algorithm](../framework/algorithm-suites.md#encryption-algorithm).
```

##### IV

The initialization vector (IV) for the frame.    
Each frame in the [Framed Data](#framed-data) MUST include an IV that is unique within the message.
The IV length MUST be equal to the IV length of the algorithm suite specified by the [Algorithm Suite ID](message-header.md#algorithm-suite-id) field.
Note: This IV is different from the [Header IV](message-header.md#iv).  

##### Encrypted Content

> The encrypted data for each frame, as returned by the [encryption algorithm](../framework/algorithm-suites.md).  

Reviewer: ```suggestion
The encrypted data for each frame, as returned by the [encryption algorithm](../framework/algorithm-suites.md#encryption-algorithm).  
```

The following describes the fields that form the Final Frame Body Structure.    
The bytes are appended in the order shown.

Field                                          | Length (bytes)                                                                                               | Interpreted as
---------------------------------------------- | ------------------------------------------------------------------------------------------------------------ | --------------
[Sequence Number End](#sequence-number-end)    | 4                                                                                                            | Bytes
[Sequence Number](#sequence-number)            | 4                                                                                                            | UInt32
[IV](#iv)                                      | [IV Length](message-header.md#iv-length)                                                                     | Bytes

> [Encrypted Content Length](#encrypted-content) | 4                                                                                                            | UInt32

Reviewer: ```suggestion
[Encrypted Content Length](#encrypted-content-length-1) | 4                                                                                                            | UInt32
```

The initialization vector for the final frame.    
The IV MUST be a unique IV within the message.   

> The IV length MUST be equal to the IV length of the [algorithm](../framework/algorithm-suites.md) that generated the message. 

Reviewer: ```suggestion
The IV length MUST be equal to the IV length of the [algorithm suite](../framework/algorithm-suites.md) that generated the message. 
```

Note: This IV is different from the [Header IV](message-header.md#iv). 

##### Encrypted Content Length

The length of the encrypted content.  

##### Encrypted Content

> The encrypted data for the final frame, as returned by the [encryption algorithm](../framework/algorithm-suites.md).  

Reviewer: ```suggestion
The encrypted data for the final frame, as returned by the [encryption algorithm](../framework/algorithm-suites.md#encryption-algorithm).
```
